project/Dependencies.scala

```diff
@@ -90,23 +91,26 @@ object Dependencies {
     val protobuf       = "com.google.protobuf"              % "protobuf-java"           % V.protobuf
 
     // Scala
-    val collectorPayload = "com.snowplowanalytics" % "collector-payload-1"                % V.collectorPayload
-    val badRows          = "com.snowplowanalytics" %% "snowplow-badrows"                  % V.badRows
-    val trackerCore      = "com.snowplowanalytics" %% "snowplow-scala-tracker-core"       % V.tracker
-    val trackerEmitterId = "com.snowplowanalytics" %% "snowplow-scala-tracker-emitter-id" % V.tracker
-    val scopt            = "com.github.scopt"      %% "scopt"                             % V.scopt
-    val akkaHttp         = "com.typesafe.akka"     %% "akka-http"                         % V.akkaHttp
-    val akkaStream       = "com.typesafe.akka"     %% "akka-stream"                       % V.akka
-    val akkaSlf4j        = "com.typesafe.akka"     %% "akka-slf4j"                        % V.akka
-    val pureconfig       = "com.github.pureconfig" %% "pureconfig"                        % V.pureconfig
-    val akkaHttpMetrics  = "fr.davit"              %% "akka-http-metrics-datadog"         % V.akkaHttpMetrics
-    val log4cats         = "org.typelevel"         %% "log4cats-slf4j"                    % V.log4cats
+    val collectorPayload  = "com.snowplowanalytics" % "collector-payload-1"                    % V.collectorPayload
+    val badRows           = "com.snowplowanalytics" %% "snowplow-badrows"                      % V.badRows
+    val trackerCore1      = "com.snowplowanalytics" %% "snowplow-scala-tracker-core"           % V.tracker1
+    val trackerEmitterId1 = "com.snowplowanalytics" %% "snowplow-scala-tracker-emitter-id"     % V.tracker1
```

Comment:
What about moving them to [`Legacy`](https://github.com/snowplow/stream-collector/blob/feature/http4s/project/Dependencies.scala#L65) as well (after renaming it) ?